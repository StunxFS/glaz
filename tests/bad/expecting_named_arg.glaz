fn x(a: i32 = 10) {}

fn main() {
  x(a=10, 1);
}
