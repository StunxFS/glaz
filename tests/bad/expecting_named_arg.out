tests/bad/expecting_named_arg.glaz:4:12: error: expected named argument
     | 
   4 |   x(a=10, 1);
     |           ^
glaz error: could not compile `expecting_named_arg`, aborting due to previous error.
